issue_template.md

```diff
@@ -44,15 +44,15 @@ paste the results here.
 ```
 If you have access to your command line run e.g.:
 sudo -u www-data php occ app:list
-from within your ownCloud installation folder
+from within your installation folder
 ```
 
 **The content of config/config.php:**
 
 ```
 If you have access to your command line run e.g.:
 sudo -u www-data php occ config:list system
-from within your ownCloud installation folder
+from within your server installation folder
```

Comment:
I think we should call it here, and in every other line (47, 74, ...) instance folder 😕 What do you think? @georgehrke